worldcup/match_getter.py

```diff
@@ -126,5 +118,5 @@ def populate_and_update(league, k=1, current_date=None):
     """
     current_date = current_date or utc_to_beijing(datetime.datetime.utcnow())
     for day_diff in range(-1, k + 1):
-        populate_match(league, current_date + datetime.timedelta(days=day_diff))
+        populate_match(league, weight_schedule, current_date + datetime.timedelta(days=day_diff))
```

Comment:
尽量用 keyword argument 传参，尤其传入的参数比较复杂或者名字对不上的时候，看代码更清楚

```suggestion
        populate_match(league, weight_schedule, date=current_date + datetime.timedelta(days=day_diff))
```